src/data/faq.json

```diff
@@ -535,6 +535,14 @@
                             "The <a href='https://github.com/corona-warn-app/cwa-documentation/blob/master/solution_architecture.md' target='_blank' rel='noopener noreferrer'>Solution Architecture Document</a> describes the technical process in detail."
                         ]
                     },
+                    {
+                        "title": "Can I enter a test result from a Corona rapid test (Schnelltest)?",
+                        "anchor": "rapid_test",
+                        "active": true,
+                        "textblock": [
+                            "No, entering results from rapid tests is currently not supported by the Corona Warn App."

```

Comment:
Corona Warn App => Corona-Warn-App

In other places in the FAQ the name is used by hyphens. 
Also in the English language version of the Google App Store
https://play.google.com/store/apps/details?id=de.rki.coronawarnapp&hl=en the name uses hyphens.

Reply:
fixed and added a reminder to #524 so that the spelling for other entries will be fixed